daemon/src/maker_inc_connections.rs

```diff
@@ -107,7 +105,26 @@ pub struct Actor {
     setup_actors: AddressMap<OrderId, setup_maker::Actor>,
     settlement_actors: AddressMap<OrderId, collab_settlement_maker::Actor>,
     rollover_actors: AddressMap<OrderId, rollover_maker::Actor>,
-    connection_tasks: HashMap<Identity, Tasks>,
+}
+
+/// A connection to a taker.
+struct Connection {
+    taker: Identity,
+    write: wire::Write<wire::TakerToMaker, wire::MakerToTaker>,
+    _tasks: Tasks,
+}
+
+impl Connection {
+    async fn send(&mut self, msg: wire::MakerToTaker) -> Result<()> {
+        let msg_str = msg.to_string();
```

Comment:
Shall we add a trace here to log what message is being sent?

Reply:
Addressed in https://github.com/itchysats/itchysats/pull/927/commits/21b20f6fdbff870393cd2f92a9d2caee9bfda50c.